resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -384,39 +384,51 @@ sub median {
     }
 }
 
-if($json{'tindaSomaticAfterRescue'} > 0) {
+if ($json{'tindaSomaticAfterRescue'} > 0) {
   $json{'tindaSomaticAfterRescueMedianAlleleFreqInControl'} = median(@SomaticRescue_control_AF);
 } else {
   $json{'tindaSomaticAfterRescueMedianAlleleFreqInControl'} = 0;
 }
 
 #######################################
 ### Annovar annotations
```

Comment:
Thanks for updates here, FYI these annovar part is removed in the hg38 branch which is waiting for the merge.